connect_ext_test.go

```diff
@@ -690,13 +690,16 @@ func TestGRPCMarshalStatusError(t *testing.T) {
 
 	mux := http.NewServeMux()
 	mux.Handle(pingv1connect.NewPingServiceHandler(
-		pingServer{},
+		pingServer{
+			// Include error details in the response, so that the Status protobuf will be marshaled.
+			includeErrorDetails: true,
+		},
+		// We're using a codec that will fail to marshal the Status protobuf, which means the returned error will be ignored
 		connect.WithCodec(failCodec{}),
 	))
 	server := memhttptest.NewServer(t, mux)
 
 	assertInternalError := func(tb testing.TB, opts ...connect.ClientOption) {
-		tb.Helper()
```

Comment:
Why remove? I suspect the linter CI job isn't going to like this change.

Reply:
I suspect removed to help debug the failing test as the helper will obfuscate which line it fails on. We do need to readd this here though.

Reply:
The linter wants this because you typically want to know which call to `assertInternalError` failed. As far as which assertion _inside the helper_ fails, it is often adequate to provide more context with a custom message.

But if this is really preferable (i.e. the line number inside the helper is more useful than the caller's line number), then add `//nolint:thelper`.